src/frontend/data/appData.ts

```diff
@@ -10,7 +10,14 @@ export const appData = {
   ),
 };
 
-// Jwt is lazily decoded only when needed
+// Jwt is lazily decoded only when needed and available
 let cacheDecodedJwt: DecodedJwt;
-export const getDecodedJwt = () =>
-  cacheDecodedJwt || (cacheDecodedJwt = jwt_decode(appData.jwt));
+export const getDecodedJwt = () => {
+  if (appData.jwt) {
+    return cacheDecodedJwt || (cacheDecodedJwt = jwt_decode(appData.jwt));
+  }
+
+  throw new Error(
```

Comment:
Pass this to `handle`? We still don't have Raven.js set up 😅 but this is something we want to know about

Reply:
I don't throw the error but use our error handler ?

Reply:
In fact I need this error to be thrown, because I'm using it in the index to manage the locale.

Reply:
If you really need the error, you should probably do both?

Reply:
In fact not having the jwt token here can be "normal" so recording it in the error handler is not a good idea IMO. I should record in the error handler where I use the function and I should have the jwt token (like in `createPlyrPlayer` for example)